### ISSUE TITLE
test

### ISSUE THREAD

**Issue by yuwwang123** [yuwwang123/f1-10_RRT_planning#1]
![image](https://user-images.githubusercontent.com/57539328/77268159-68029b80-6c7b-11ea-9c03-b56e23776ce0.png)

**Comment by biagiolicari** [comment]
Hi, i'm work ti create and rrt* for F1/10 but i would ti try your implementation ti see how It work First....how can i try this package ? Thanks so much

**Comment by yuwwang123** [comment]
Have installed the f1/10 racecar simulator in Rviz? (https://github.com/mlab-upenn/f110-fall2019-skeletons/tree/master/racecar_simulator).

If you do, then just clone my package and do a rosrun

**Comment by biagiolicari** [comment]
Yes, I've installed f1-10 racecar simulator but when I do a rosrun this is what appears to me
![2](https://user-images.githubusercontent.com/3578517/81331624-29496900-90a2-11ea-8ca5-db6ea941dc61.png)
![1](https://user-images.githubusercontent.com/3578517/81331631-2a7a9600-90a2-11ea-8d39-2d09e54c492e.png)

and I can't set a goal. Thanks for your time and sorry for all these questions but I'm new in this world

**Comment by yuwwang123** [comment]
I forgot to include the waypoints file in the repo. Now it's been added. Just download the waypoints.csv file and replace the path in rrt.cpp to wherever you store the file, then you should be good. This RRT planner is not a global one. At each timestep it only plans a path to a goal that is a certain distance ahead, but the goal keeps getting advanced as the car moves, so basically the car is just gonna keep driving around the corridor.  (You will still see the "couldn't find a path" prompt sometimes since it's not gauranteed to find a path every time)

**Comment by biagiolicari** [comment]
Sorry ti disturb you again, if I want a safer path distant from the obstacles or the edge of the map, is it possible by changing the value of some variable?

**Comment by grljjj** [comment]
Hi, can you help me with launching this project please. I have f1tenth_simulator. I launch f1tenth_simulator with launch file, then in other terminal window rosrun f1-10_RRT_planning rrt_node. After that i can see only fast changing dots in rviz simulation, car doesnt move, no trees. 
P.S. I soured CSV in rrt.cpp
What should i do?

**Comment by yuwwang123** [comment]
I think you need to select the topics to view in rviz by checking the boxes in topic list. The topic for tree is "tree branch" if I remembered correctly. The moving dots you saw are the tree nodes

**Comment by yuwwang123** [comment]
> Sorry ti disturb you again, if I want a safer path distant from the obstacles or the edge of the map, is it possible by changing the value of some variable?

Yeah I think you can change how much you want to inflate the obstacles

**Comment by grljjj** [comment]
Thank you for answer! But how can i start simulation with rrt, car doesnt move?

**Comment by yuwwang123** [comment]
> Thank you for answer! But how can i start simulation with rrt, car doesnt move?

You need to press n to unlock the car I think (details can be found on the f1tenth simulator repo page)
